mindmeld/augmentation.py

```diff
@@ -204,6 +236,12 @@ def __init__(self, batch_size, language, paths, path_suffix, resource_loader):
             path_suffix (str): Suffix to be added to new augmented files.
             resource_loader (object): Resource Loader object for the application.
         """
+
+        if language != 'en':
```

Comment:
Nit: Let's use the ENGLISH_LANGUAGE_CODE global variable from components._config instead of 'en'